tools/functions.sh

```diff
@@ -1,4 +1,4 @@
-# shellcheck shell=sh
+#!/bin/bash
```

Comment:
Not here. This script is not meant to be executed directly, so it shouldn't have a shebang line.
```suggestion
# shellcheck shell=bash
```